probe_scraper/runner.py

```diff
@@ -101,6 +101,13 @@ def write_moz_central_probe_data(probe_data, revisions, out_dir):
 
 def write_general_data(out_dir):
     dump_json(general_data(), out_dir, "general")
+    open(os.path.join(out_dir, "index.html"), 'w').write(
+        "<html><head><title>Mozilla Probe Info</title></head>"
+        "<body>This site contains metadata used by Mozilla's data collection "
+        "infrastructure, for more information see "
+        "<a href=\"https://mozilla.github.io/probe-scraper/\">the generated documentation</a>."
+        "</body></html>"
+    )
```

Comment:
To be more idiomatic, I'd like to see this expressed as a context manager and a multiline string:

```
with open(os.path.join(out_dir, "index.html"), 'w') as f:
    f.write("""
    """)
```

I don't see any particular reason to worry about indentation in the output file.

Reply:
Just realized I only half fixed this, will do the other half before landing.